kernelci/api/models.py

```diff
@@ -385,6 +385,9 @@ class KbuildData(BaseModel):
     kernel_type: Optional[str] = Field(
         description="Kernel image type (zimage, bzimage...)"
     )
+    regression: Optional[PyObjectId] = Field(
+        description="Regression node related to this build instance"
+    )
```

Comment:
We are already storing node IDs that caused regressions to `Regression` node. And that is enough to link failing nodes with regression. Then why are we storing regression ID here as well?

Reply:
Okay. So basically this bi-directional link is for querying regressions.

> For the second case we can't because we can't do a query like "find regressions whose node_sequence contain this node ID"

In that case, this worked:
```
$ curl https://staging.kernelci.org:9000/latest/nodes?data.node_sequence=6603a4d35ca93487c2da3c76
```
The equality `=` operator works for querying an array field as well. See https://www.mongodb.com/docs/v3.4/tutorial/query-arrays/#query-an-array-for-an-element

Reply:
Yes, would we still need this `regression` field?